### ISSUE TITLE
Support usrloc federated cluster without NoSQL(mongodb/cassandra/couchdb)

### ISSUE THREAD

**Issue by truong-hua** [OpenSIPS/opensips#2908]
**Is your feature request related to a problem? Please describe.**

Nowaday SQL is scalable like YugabyeDB or Cockroachdb, so there is no need to maintain another NoSQL DB to implement Federated Cluster. Which will help to simplify the deployment of OpenSIPS but still keep the scalability.

**Describe the solution you'd like**
Support columnar capability to cachedb_sql or a way to config federated cluster without enabled cachedb.

**Describe alternatives you've considered**
No any alternatives unless deploying both SQL and NoSQL.

**Comment by liviuchircu** [comment]
Yes, there are so many scalable and popular SQL solutions nowadays (Redis Cluster, PXC, YugaByte, etc.), that we should definitely add support for the federated cluster to work with them.

Nowadays, with the current solution (originally designed in 2018), you most often end up maintaining an additional NoSQL cluster just for storing user location metadata, which although seems nice from a separation (microservices) point of view, the load on this cluster is not that high and it could very well work with just an extra database in your SQL cluster, e.g. `user_location`.
